lib/ash/actions/update/bulk.ex

```diff
@@ -404,14 +404,9 @@ defmodule Ash.Actions.Update.Bulk do
       raise ArgumentError, "Cannot specify `sorted?: true` and `return_stream?: true` together"
     end
 
-    {context_key, metadata_key} =
-      case action.type do
-        :update ->
-          {:bulk_update, :bulk_update_index}
-
-        :destroy ->
-          {:bulk_destroy, :bulk_destroy_index}
-      end
+    context_key = :bulk_update
```

Comment:
We still need to switch on `action.type` so that this uses the action type for these three values. For destroys the context key, metadata key, and ref_metadata_key need to be based on the action type.

Reply:
this one I was less sure about, but I think you mean:

```diff
-    context_key = :bulk_update
-    metadata_key = :bulk_update_index
-    ref_metadata_key = :bulk_action_ref
+    {context_key, metadata_key, ref_metadata_key} =
+      case action.type do
+        :destroy ->
+          {:bulk_destroy, :bulk_destroy_index, :bulk_action_ref}
+
+        _ ->
+          {:bulk_update, :bulk_update_index, :bulk_action_ref}
+      end
```

Correct?